pkg/docker/config/config_test.go

```diff
@@ -36,20 +37,27 @@ func TestGetPathToAuth(t *testing.T) {
 		// Default paths
 		{&types.SystemContext{}, linux, "", "/run/containers/" + uid + "/auth.json", false, false},
 		{&types.SystemContext{}, darwin, "", darwinDefault, false, false},
+		{&types.SystemContext{}, freebsd, "", darwinDefault, false, false},
 		{nil, linux, "", "/run/containers/" + uid + "/auth.json", false, false},
 		{nil, darwin, "", darwinDefault, false, false},
+		{nil, freebsd, "", darwinDefault, false, false},
 		// SystemContext overrides
 		{&types.SystemContext{AuthFilePath: "/absolute/path"}, linux, "", "/absolute/path", false, true},
 		{&types.SystemContext{AuthFilePath: "/absolute/path"}, darwin, "", "/absolute/path", false, true},
+		{&types.SystemContext{AuthFilePath: "/absolute/path"}, freebsd, "", "/absolute/path", false, true},
 		{&types.SystemContext{LegacyFormatAuthFilePath: "/absolute/path"}, linux, "", "/absolute/path", true, true},
 		{&types.SystemContext{LegacyFormatAuthFilePath: "/absolute/path"}, darwin, "", "/absolute/path", true, true},
+		{&types.SystemContext{LegacyFormatAuthFilePath: "/absolute/path"}, freebsd, "", "/absolute/path", true, true},
 		{&types.SystemContext{RootForImplicitAbsolutePaths: "/prefix"}, linux, "", "/prefix/run/containers/" + uid + "/auth.json", false, false},
 		{&types.SystemContext{RootForImplicitAbsolutePaths: "/prefix"}, darwin, "", "/prefix/run/containers/" + uid + "/auth.json", false, false},
+		{&types.SystemContext{RootForImplicitAbsolutePaths: "/prefix"}, freebsd, "", "/prefix/run/containers/" + uid + "/auth.json", false, false},
```

Comment:
Actually, I think these should be just `darwinDefault`
- `RootFor…` is ~chroot-like, it should not cause a switch to Linux semantics
- `RootFor…` is documented not to affect paths in `$HOME`.

(I guess that feature is extremely rarely, if ever, used, so no-one has noticed in all those years.)